community/audio/readme.txt

```diff
@@ -1 +1 @@
-For music download .ogg or other from https://archive.org/details/C418-MinecraftSoundtrackVolumeAlpha/ and put them into the music dir
\ No newline at end of file
+For music download .ogg (or other) from https://archive.org/details/C418-MinecraftSoundtrackVolumeAlpha/ and put them into the music dir
```

Comment:
Could you rename `readme.txt` -> `README.md`?

Also add backticks around `.ogg` and make the URL a proper.